packages/kolibri-tools/lib/cli.js

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/env node
 
+require('colors');
```

Comment:
Can drop this and just use the `cliLogging.error` function in place of `console.error` with explicit colouring applied.